lib/service/storage/elasticsearch.ts

```diff
@@ -2205,10 +2387,18 @@ class ElasticSearch extends Service {
    * @returns {Promise.<Object>} { items, errors }
    */
   async mCreate(
-    index,
-    collection,
-    documents,
-    { refresh, timeout, userId = null } = {},
+    index: string,
+    collection: string,
+    documents: JSON[],
+    {
+      refresh,
+      timeout,
+      userId = null,
+    }: {
+      refresh?: boolean | "wait_for";
+      timeout?: string;
+      userId?: string;
+    } = {},
```

Comment:
```suggestion
    {
      refresh,
      timeout,
      userId = null,
    }: Exclude<KRequestParams, "injectKuzzleMeta" | "retryOnConflict"> = {},
```

With new `KRequestParams` type we can compose type like it, to DRY